Assets/Scripts/UI/Pickup Pup/DogProfile/DogProfile.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+ * Author(s): Grace Barrett-Snyder 
+ * Description: Displays a Dog's Profile from its DogDescriptor
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DogProfile : PPUIElement
+{
+    [SerializeField]
+    Text nameText;
+    [SerializeField]
+    Text breedText;
+    [SerializeField]
+    Text[] descriptionText;
+
+    [SerializeField]
+    Image dogThumbnail;
+    [SerializeField]
+    Image collarImage;
+
+    [SerializeField]
+    GameObject leftArrow;
+    [SerializeField]
+    GameObject rightArrow;
+
+    [SerializeField]
+    UIButton closeWindowHitArea;
+
+    [SerializeField]
+    protected Button rehomeButton;
+    [SerializeField]
+    protected Button collarSlot;
+
+    protected DogDescriptor dogInfo;
+
+    #region MonoBehaviourExtended Overrides
+
+    protected override void subscribeEvents()
+    {
+        base.subscribeEvents();
+        closeWindowHitArea.SubscribeToClick(Hide);
+    }
+
+    #endregion
+
+    public virtual void SetProfile(Dog dog)
+    {
+        dogInfo = dog.Info;
+
+        nameText.text = dogInfo.Name;
+        breedText.text = dogInfo.Breed.Breed;
+        for(int i = 0; i < dogInfo.Descriptions.Length; i++)
```

Comment:
Could cause an `IndexOutOfRange` error if the descriptions array is longer than the text arr. Maybe better to add text objects dynamically at runtime? Or do range checks.